initScripts/aarch32/Ubuntu_16.04/Docker_18.03.sh

```diff
@@ -373,6 +376,22 @@ fetch_cexec() {
   popd
 }
 
+enable_32_bit_builds() {
+  __process_marker "Adding files to run 32-bit builds"
+
+  local temp_file="/tmp/copy-file.sh"
+  local base_docker_image="arm32v7/ubuntu:16.04"
+
+  rm -rf /tmp/* || true
```

Comment:
This doesn't seem like a good idea to me. There may be other things on the node that are actively using tmp while we're initialization.